packages/webapp/src/lib/stores/settings.ts

```diff
@@ -114,11 +107,12 @@ export const activeSubgraphs = cachedWritableStore<Record<string, SubgraphCfg>>(
  * @default "" - Empty string by default
  * @returns A writable store containing the order hash string
  */
-export const orderHash = cachedWritableStore<string>(
+export const orderHash = cachedWritableStore<Hex>(
```

Comment:
as in other comment - what is this for?

Reply:
this is for the order hash filter in orders list table page